src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableClient.cs

```diff
@@ -484,6 +484,40 @@ async Task<DurableOrchestrationStatus> IDurableOrchestrationClient.GetStatusAsyn
             return await GetDurableOrchestrationStatusAsync(state, this.client, showHistory, showHistoryOutput);
         }
 
+        /// <inheritdoc />
+        async Task<IList<DurableOrchestrationStatus>> IDurableOrchestrationClient.GetStatusAsync(IEnumerable<string> instanceID, bool showHistory, bool showHistoryOutput, bool showInput)
+        {
+            IList<DurableOrchestrationStatus> durableorchestrationstateList = new List<DurableOrchestrationStatus>();
+            foreach (string id in instanceID)
+            {
+                IList<OrchestrationState> stateList = null;
+                try
+                {
+                    stateList = await this.DurabilityProvider.GetOrchestrationStateWithInputsAsync(id, showInput);
+                }
+                catch (NotImplementedException)
+                {
+                    // TODO: Ignore the show input flag. Should consider logging a warning.
+                    stateList = await this.client.ServiceClient.GetOrchestrationStateAsync(id, allExecutions: false);
+                }
+
+                OrchestrationState state = stateList?.FirstOrDefault();
+                DurableOrchestrationStatus currentidstate = new DurableOrchestrationStatus();
```

Comment:
This is a confusing variable name and it doesn't follow C# naming guidelines. Here's a good reference for C# naming guidelines: [link](https://github.com/ktaranov/naming-convention/blob/master/C%23%20Coding%20Standards%20and%20Naming%20Conventions.md#c-coding-standards-and-naming-conventions).

Here's my suggestion:
```suggestion
                var currentStatus = new DurableOrchestrationStatus();
```